plugins/BEdita/Core/src/Model/Validation/ObjectsValidator.php

```diff
@@ -57,8 +57,10 @@ public function __construct()
             ->allowEmptyString('title')
 
             ->allowEmptyString('description')
+            ->maxLengthBytes('description', (2 << 23) - 1) // 2^24 - 1 bytes = 16MiB
```

Comment:
Rather than hardcoding these values, could we read length from cached schema?

```suggestion
            ->maxLengthBytes('description', $this->getSchema()->getColumn('description')['length'])
```

Reply:
The schema's length for sqlite and Postgress is empty, so I used `MysqlAdapter::TEXT_MEDIUM` which is a numeric costant